src/i18n/es.js

```diff
@@ -62,7 +62,7 @@ export default {
   forgotPasswordTitle: 'Restablecer contraseña',
   forgotPasswordAction: '¿Olvidó su contraseña?',
   forgotPasswordInstructions:
-    'Por favor ingrese su dirección de correo. Le enviaremos las instrucciones para restablecer su contrseña.',
+    'Por favor escriba su dirección de correo. Le enviaremos las instrucciones para restablecer su contrseña.',
```

Comment:
`ingrese` is used in the rest of the sentences of this same file. I would keep it.